samples/net/zoap_server/prj.conf

```diff
@@ -9,3 +9,4 @@ CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_ZOAP=y
 CONFIG_NET_APP_SETTINGS=y
 CONFIG_NET_APP_MY_IPV6_ADDR="2001:db8::1"
+CONFIG_NET_IF_MCAST_IPV6_ADDR_COUNT=5
```

Comment:
Oh, and btw, we can follow "static assert" approach (#if/#error in this case), that should do just what we need.